### PR TITLE
Bug 1162040 - updating accessibility for gaia-icons used in system app.

### PR DESCRIPTION
**Pull request by yzen**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=357da45d3bc26c684c160a1ae7a78ae59788429f

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ee3fc7796758fdcf7954b0b840ada5d42c847813

File: apps/system/js/quick_settings.js [hunk 1]
```diff
@@ -194,8 +194,9 @@
       var wifiFirstSet = true;
       SettingsListener.observe('wifi.enabled', true, function(value) {
         // check self.wifi.dataset.enabled and value are identical
-        if ((self.wifi.dataset.enabled && value) ||
-          (self.wifi.dataset.enabled === undefined && !value)) {
+        if ((self.wifi.dataset.enabled === 'true' && value) ||
+          ((self.wifi.dataset.enabled === undefined ||
+            self.wifi.dataset.enabled === 'false') && !value)) {
```
Comment (mikehenrty) [inline apps/system/js/quick_settings.js]: :+1:

File: apps/system/js/media_recording.js [hunk 2]
```diff
@@ -159,7 +159,7 @@
           messageElement, timerElement;
       /* create panel
        <div class="media-recording-status fake-notification" role="listitem">
-         <div data-icon="video"></div>
+         <div data-icon="video" aria=hidden="true"></div>
```
Comment (mikehenrty) [inline apps/system/js/media_recording.js]: should be `area-hidden="true" not area**=**hidden="true"

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=218091a21855efff9252e4cbc6e5b29a9353cdb9

**Pull request merged**